sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Scenario.Tests/tests/Microsoft.Azure.WebJobs.Extensions.Storage.Scenario.Tests.csproj

```diff
@@ -5,8 +5,9 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" VersionOverride="2.14.0" />
-    <PackageReference Include="Microsoft.AspNetCore" VersionOverride="2.1.0" />
-    <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" VersionOverride="2.1.0" />
+    <PackageReference Include="Microsoft.AspNetCore" VersionOverride="2.1.7" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" VersionOverride="2.1.3" />
+    <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel.Core" VersionOverride="2.1.25" />
```

Comment:
Why is Kestrel being added here?

Reply:
In this particular case, it's upgrading an indirect dependency. Version 2.1.0 has a CVE, and this was the most recent 2.1.x. I did update Microsoft.AspNetCore.* globally for most other projects, but those were already using 2.2.x. For security updates, we do the same as .NET: update to the latest patch version, if available.

If you're the owner of this project and are fine with updating to 2.2.x I can do so instead.

Reply:
If tests pass then 2.2.x should be fine. let's try that.

Reply:
I updated the references. If tests pass I'll merge.